src/SFA.DAS.ContentApi.Database/Tables/Client.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Client]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [Description] VARCHAR(50) NULL, 
+    [Identity] NVARCHAR(20) NOT NULL, 
+    CONSTRAINT [IX_Client_Code] UNIQUE ([Identity])
```

Comment:
Why does the unique constraint have the name Client_Code for the Identity column, is the column misnamed, the index misnamed or does it convey some meaning?